### PR TITLE
fix: adjusted styling of language select across browsers

### PR DESCRIPTION
**Pull request by raphael-arce**
- adjusted styling of language select across browsers
- adjusted styling of home button
- added e2e test
- added some comments in the `index.css` to explain why these styles are there

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206584356611903

### CODE REVIEW

File: src/index.css
Comment (raphael-arce) [inline src/index.css:1]: @aeschi i'm not quite sure if the comments are well formulated and easy to understand. what do you think?

Comment (aeschi) [reply src/index.css:1]: @raphael-arce I think they are understandable 👍

**Review by aeschi — CHANGES_REQUESTED** [review]
The language selector still looks a bit off (too wide) on safari and chrome on my phone.

**Comment by raphael-arce** [comment]
Okay, will look into it! 👍

**Comment by raphael-arce** [comment]
@aeschi after some research I think we cannot change styling of the box that gets opened on iOS, as it is defined by the operating system and there is apparently no way of changing that. This box unfortunately looks different in most browsers/OS: https://css-tricks.com/dropdown-default-styling/#aa-what-about-the-dropdown-itself

We could use a different element than the standard `<select>` and use some custom implementation to achieve a similar select/dropdown functionality, allowing full freedom of design. But I think for this situation it is better to stick to the standard. What do you think?

**Comment by aeschi** [comment]
> @aeschi after some research I think we cannot change styling of the box that gets opened on iOS, as it is defined by the operating system and there is apparently no way of changing that. This box unfortunately looks different in most browsers/OS: https://css-tricks.com/dropdown-default-styling/#aa-what-about-the-dropdown-itself
> 
> We could use a different element than the standard `<select>` and use some custom implementation to achieve a similar select/dropdown functionality, allowing full freedom of design. But I think for this situation it is better to stick to the standard. What do you think?

ah interesting! Ok, then let's stickt with the standard.

**Pull request merged**